server/game/gameSystems/DistributeExperienceSystem.ts

```diff
@@ -22,6 +22,6 @@ export class DistributeExperienceSystem<TContext extends AbilityContext = Abilit
     }
 
     protected override canDistributeLessDefault(): boolean {
-        return this.properties.canDistributeLess;
+        return this.properties.canDistributeLess || false;
```

Comment:
Still doesn't match the other implementations of this method, and it doesn't really make sense for it to check the properties since it's supposed to be providing the default in case the property isn't present. Also, it's checking `this.properties` instead of using `generatePropertiesFromContext`, so it might be getting a wrong value. Should just return `false`